maven-repository-manager/pom.xml

```diff
@@ -47,6 +47,10 @@
       <artifactId>spi</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
it's in provided scope, in dependencyManagement below. It may make sense to put it in test scope, but I'm not 100% sure. Provided scope will not propagate into the dependency graph though.